data/fleet-commander-standalone.in

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ $UID != 0 ]; then
+    echo "WARNING: You need to be root in order to have access to dbus service"
```

Comment:
I am missing a warning in case we run with "localhost" as HOST to let the user know that the live session won't work in this case